static/src/js/util/createSpatialDisplay.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Returns a string displaying the spatial information.
+ * @param {Object} spatial Object that holds the different spatial areas.
+ * @returns {String} Returns a string formatting the spatial areas into human readable values.
+ */
+export const createSpatialDisplay = (spatial) => {
+  const {
+    boundingBox,
+    circle,
+    line,
+    point,
+    polygon
+  } = spatial
+
+  const selectedShape = boundingBox || circle || line || point || polygon
+
+  if (selectedShape) {
+    const splitStr = selectedShape[0].split(',')
+
+    if (boundingBox) { return `SW: (${splitStr[1]}, ${splitStr[0]}) NE: (${splitStr[3]}, ${splitStr[2]})` }
```

Comment:
Already approved but just caught this. Some new lines after each of this `if`s might help readability.